bin/generate_saml_pki

```diff
@@ -1,2 +0,0 @@
-#!/bin/sh
-openssl req -x509 -sha256 -days 365 -newkey rsa:2048 -keyout keys/saml.key.enc.example -out certs/saml.crt.example -config config/openssl.conf
```

Comment:
same thing lives in https://developers.login.gov/testing/#creating-a-public-certificate